test/connectivity/scripts/testHelper.sh

```diff
@@ -117,3 +131,10 @@ function print_highlighted_message() {
     local color_reset='\033[0m'
     echo -e "${cyan}$message${color_reset}"
 }
+
+function stop_iotedge_service() {
+    echo 'Stop IoT Edge services'
+    systemctl stop iotedge.socket iotedge.mgmt.socket || true
+    systemctl kill iotedge || true
+    systemctl stop iotedge || true
+}
```

Comment:
Why can't you only do ```systemctl stop iotedge```?

Reply:
I just refactor and extract the original implementation.  We can discuss with Arnav and handle it in different PR